pkg/controllers/state/statenode.go

```diff
@@ -195,9 +195,6 @@ func (in *StateNode) ValidateNodeDisruptable(ctx context.Context, kubeClient cli
 	}
 	// check whether the node has all the labels we need
 	for _, label := range []string{
-		v1.CapacityTypeLabelKey,
```

Comment:
Why remove these labels?

Reply:
We don't need to check for these to see if a node is explicitly disruptable. It only matters for Consolidation now and is done in the shouldDisrupt step